docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -1,66 +1,129 @@
 # Configurations for Intel® Processor Graphics (GPU) with OpenVINO™ {#openvino_docs_install_guides_configurations_for_intel_gpu}
 
-
 @sphinxdirective
 
 .. _gpu guide:
 
-
-To use the OpenVINO™ GPU plugin and offload inference to Intel® Processor Graphics (GPU), Intel® Graphics Driver must be properly configured on your system.
-
-If Intel® Graphics Driver is already installed and you would like to keep it, you can skip the installation steps below.
+To use the OpenVINO™ GPU plug-in and transfer the inference to the graphics of the Intel® processor (GPU), the Intel® graphics driver must be properly configured on the system.
 
 Linux
 #####
 
-To install the latest available **Intel® Graphics Compute Runtime for oneAPI Level Zero and OpenCL™ Driver** for your operating system, 
-see its `installation guide on GitHub <https://github.com/intel/compute-runtime/releases/latest>`_.
+To use a GPU device for OpenVINO inference, you must meet the following prerequisites:
 
-.. note::
+- use a supported Linux kernel as per the `documentation <https://dgpu-docs.intel.com/driver/kernel-driver-types.html>`__
+- install GPU Runtime drivers: `The Intel(R) Graphics Compute Runtime for oneAPI Level Zero and OpenCL(TM) Driver <https://github.com/intel/compute-runtime/releases/latest>`__
+- install `Intel Graphics Memory Management Library <https://github.com/intel/gmmlib>`__ and `Intel® Graphics Compiler for OpenCL™ <https://github.com/intel/intel-graphics-compiler>`__
+- install `OpenCL ICD loader package <https://github.com/KhronosGroup/OpenCL-ICD-Loader>`__
 
-  If you are using RedHat 8, you can install the OpenCL library as a prerequisite by using the following command: 
-  ``http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/ocl-icd-2.2.12-1.el8.x86_64.rpm``
+Depending on your operating system, there may be different methods to install the above packages. Below are the instructions on how to install the packages on supported Linux distributions.
 
+Ubuntu 22.04 LTS
+++++++++++++++++
```

Comment:
should we turn it into tabs? So, I can select between Ubuntu 22, Ubuntu 20 or UBI 8